doc/1.4/language.md

```diff
@@ -2935,6 +2939,36 @@ scope.
 Note that initializers of such variables are evaluated *once* when
 initializing the device, and thus must be a compile-time constant.
 
+
+### Return Statements
+<pre>
+return [<em>initializer</em>];
+</pre>
+
+Returns from method with the value(s) specified by the argument.
+Unlike C, the argument is an *initializer*, meaning, for example, return
+values of struct-like type can be constructed using <tt>{<em>...</em>}</tt>.
+
+The initializer must provide the exact number of values corresponding as the
+return values of the method, and they must be of compatible type. Multiple
+values can be provided either through:
+* Providing an initializer for each return value through tuple syntax, e.g.:
+```
+method m() -> (bool, int) {
+    return (false, 4);
+}
+```
+* Performing a method call and propagating the return values:
+```
+method n(int i) -> (bool, int) {
+    ...
+}
+
+method m() -> (bool, int) {
+    return n(4);
+}
+```
```

Comment:
you could make this an extension of the previous example and flip the relation, `method n() -> (bool, int) { return m(); }`.